unit_testing/test_labels.py

```diff
@@ -113,17 +119,19 @@ def test_increment_z_inverse(test_image):
     assert diff.all()
 
 
-@pytest.mark.parametrize("test_seg,test_labels", [(seg_img, labels_img)])
-def test_labelize_from_discs(test_seg, test_labels):
-    seg = test_seg.copy()
-    ref = test_labels.copy()
+@parametrize_with_cases("seg_img", cases=[seg_img])
+@parametrize_with_cases("labels_img", cases=[labels_img])
```

Comment:
careful here: each @parametrize gets *multiplied* in -- it does a big cross product. You can't tell here because you've only got one each but if you added any others you'd see you need to keep this all in a single line or else you'll be mixing inputs with mismatched outputs.

Reply:
Maybe this one shouldn't even be parametrized, since it's only testing one case.